internal/broadcast/types.go

```diff
@@ -91,3 +91,15 @@ type WorkstationSeen struct {
 	Hostname string
 	LastSeen int64
 }
+
+type HistoricalData []HistoricalDataPoint
+
+type HistoricalDataPoint struct {
+	Timestamp int64 `json:"timestamp"`
+	Samples   []GPU `json:"sample"`
+}
+
+type AggregateData struct {
+	PercentUsed int    `json:"percent_used"`
+	TotalEnergy uint64 `json:"total_energy"`
```

Comment:
Would be great to get a quick description/units we transfer energy in

Reply:
apparently, power draw is in Watts, so according to that we should be getting Joules. I will document that